zigpy/config/__init__.py

```diff
@@ -65,6 +67,7 @@
 CONF_NWK_MAX_RETRIES = "max_retries"
 CONF_NWK_TC_ADDRESS = "tc_address"
 CONF_NWK_TC_LINK_KEY = "tc_link_key"
+CONF_NWK_TX_POWER = "tx_power"
```

Comment:
note: I'm sure this is already on your radar, but zigpy-znp should probably be updated then to respect this zigpy option (if the znp specific `tx_power` option isn't set).

I'd be in favor of removing the znp specific option, but I'm not sure how many people pasted something with `tx_power: 19` in their configuration and forgot about it 😅